variables.tf

```diff
@@ -32,3 +37,23 @@ variable "vpc_endpoint" {
   default     = null
   description = "Optional VPC endpoint settings for your SFTP server"
 }
+variable "restricted_mode" {
```

Comment:
You are looping over var.users but are also using restrict_mode. Should restrict_mode not be part of the users variable? Same for the s3_id var?

Reply:
The idea here is that with a single flag, you can switch between the original mode (for backwards compatibility), as well as towards a 'chroot' like style config. This makes the user config less error prone.

Reply:
Agreed that it makes sense to make this a generic flag instead of a user-specific flag.